.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Cleanup the pr please. This shouldn't be pushed

Reply:
Sorry, i did ...what about now?